src/ResourceManager/DataFactories/Commands.DataFactoryV2.Test/ScenarioTests/IntegrationRuntimeTests.ps1

```diff
@@ -73,11 +85,6 @@ function Test-SelfHosted-IntegrationRuntime
 .SYNOPSIS
 Creates a SSIS-Azure integration runtime and then does operations.
 Deletes the created integration runtime at the end.
-
-Before running this test, make sure setting the following environment variables:
```

Comment:
@zhangyd2015 any reason for removing this comment?

Reply:
This comment is not valid anymore. We don't use environment variables passing credential.